packages/modules/notification/src/loaders/providers.ts

```diff
@@ -89,7 +88,7 @@ async function syncDatabaseProviders({
   validateProviders(normalizedProviders)
 
   try {
-    const providersInDb = await providerService.list({})
+    const providersInDb = await providerService.find()
```

Comment:
question: Why does this service have `find` and not `list`? Should be an internally generated service no?

Reply:
Ah, I guess this is because we are resolving the repo instead. We try to avoid those leaking outside of services. 

If this is absolutely necessary, I would maybe suggest we rename the `providerService` to `providerRepo`. Up to you.

Reply:
But shouldn't we be able to do `resolve("notificationProviderService")`